examples/wasm/iwasmt.c

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (C) 2019 Intel Corporation.  All rights reserved.
+ * Copyright (C) 2020 TU Bergakademie Freiberg Karl Fessel
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+#include <string.h>
```

Comment:
```suggestion
#include <stdbool.h>
#include <stdint.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#include "thread.h"
```

We usually use `<>` only for headers that are part of the standard C lib, and `""` for RIOT headers. (The `#include <thread.h>` from below can than be dropped. I also was so free to sort the headers alphabetically.)